app/forms/creator_settings_form.rb

```diff
@@ -0,0 +1,65 @@
+class CreatorSettingsForm
+  include ActiveModel::Model
+  include ActiveModel::Attributes
+
+  attribute :checked_code_of_conduct, :boolean, default: false
+  attribute :checked_terms_and_conditions, :boolean, default: false
+  attribute :community_name, :string
+  attribute :invite_only_mode, :boolean
+  attribute :logo
+  attribute :primary_brand_color_hex, :string
+  attribute :public, :boolean
+
+  validates :community_name,
+            :primary_brand_color_hex, presence: true
+
+  validates :checked_code_of_conduct, inclusion: { in: [true, false] }
+  validates :checked_terms_and_conditions, inclusion: { in: [true, false] }
+  validates :invite_only_mode, inclusion: { in: [true, false] }
+  validates :public, inclusion: { in: [true, false] }
+
+  attr_accessor :success
+
+  def initialize(attributes = {})
+    super
+
+    self.checked_code_of_conduct = checked_code_of_conduct
```

Comment:
This confused me a bit. When you put a `binding.pry` after `super`, you'll see that all attributes on the left-hand side already have the value that's on the right hand side because you're using the getter to update the setter, which means you overwrite the value with itself. 

Completely removing the `initialize` method seems to work fine (the initializer spec is still green) because we end up using the one from `ActiveModel::Attributes`:

```ruby
CreatorSettingsForm.new.method(:initialize).owner
#=> ActiveModel::Attributes
```